app/ports/tests/test_update_portinfo.py

```diff
@@ -29,7 +29,137 @@ def test_deleted(self):
         self.assertEquals(port_status_mainport, True)
         self.assertEquals(port_status_subport, False)
 
+    def test_moved(self):
+        Port.update([
+            {
+                "variants": ["universal"],
+                "portdir": "categoryA/port-A1",
+                "depends_fetch": ["bin:port-A4:port-A4"],
+                "description": "This is port A1 of categoryA",
+                "homepage": "http:\/\/portA1.website\/",
+                "epoch": "0",
+                "platforms": "darwin",
+                "name": "port-A2-subport",
```

Comment:
Just to clarify it for myself: in what scenario do you end up calling `Port.update` with just the list of two subports in the main port's path?

Reply:
I am adding two more subports to the main port 'port-A1'. This happens only for this test.